roles/vm-spinup/templates/spinup.sh.j2

```diff
@@ -26,10 +26,10 @@ DIR={{ images_directory }}
 IMAGE={{ images_directory }}/{{ image_destination_name }}
 
 # Amount of RAM in MB
-MEM={{ vm_parameters_ram_mb }}
```

Comment:
Hrmmm, one thought -- is there a way we can write this file with the system default? And then you can call `./spinup.sh host-name` or optionally `./spinup.sh hostname MEM CPU`? That way it's still easily used by the lazy humans?

Reply:
Yea, I think I know how to do this :)

Reply:
DONE!